_locales/ca.json

```diff
@@ -171,13 +171,13 @@
         "message": "E-Mail Address",
         "description": "Label above input field for e-mail address"
     },
-    "login.password": {
+    "login.mailPw": {
         "message": "Password",
-        "description": "Label above input field for password"
+        "description": "Label above input field for mailPw"
```

Comment:
This is a description. Why not just write `email password`?

Reply:
good hint :+1: